src/servico/NotificacaoService.java

```diff
@@ -86,7 +92,10 @@ public String notificarAusencia(Long idProfessor, String dataInicio, String data
         
         for(Ausencia ausencia : ausencias)
         {
-            String codigo = Integer.toString(r.nextInt(10000));
+            
+            int randomNumber = r.nextInt(10000);

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['10000' is a magic number.](https://www.codacy.com/app/calebepb/substituicao/file/2728205073/issues/source?bid=3308597&fileBranchId=3349120#l96)